### DISCUSSION TITLE
Image-button-response in cognition.run

### DISCUSSION THREAD

**Discussion by ler93** [jspsych/jsPsych#1422] [Q&A] [answered]
Hi, everyone. 

I'm trying to create a variable of type "image-button-response" with some choice options below. I was able to do it perfectly in "Visual Studio Code" but I have the problem when trying to copy the code in "cognition.run" page. I suppose it is because the page cannot find the folder that I indicated in the stimulus. I'm totally new to programming so I don't know how I can solve it. 

Here is the code: 

_var image = {
        type: 'image-button-response',
        stimulus: 'jspsych/examples/img/raven.png',
        stimulus_height: 500,
        choices: ['1', '2', '3', '4', '5', '6', '7', '8'],
    };_

Thanks in advance.

**Comment by becky-gilbert** [comment, marked as answer]
**EDIT**: I'm pretty sure my answer below is wrong. I think cognition.run ignores the stimulus file path and just looks to see if it can access the file name. [Josh's answer below](https://github.com/jspsych/jsPsych/discussions/1422#discussioncomment-316239) solved this problem.

~~Hi @ler93, your local directory structure isn't maintained on cognition.run, so if you upload any stimuli files, you can access them with just the file name. Try this in your cognition.run version:~~
```javascript
var image = {
  type: 'image-button-response',
  stimulus: 'raven.png',
  stimulus_height: 500,
  choices: ['1', '2', '3', '4', '5', '6', '7', '8'],
};
```

**Reply by ler93** [reply]
Hi, @becky-gilbert. It still doesn't work. The message that I see on the screen is "The experiment failed to load." 
I'm sure that is a problem in the variable "image" because when I remove it, the experiment works fine.

Here is the link: https://c8xrwjk7qn.cognition.run/

Could there be any other solution?

Thanks!

**Reply by jodeleeuw** [reply]
Hi @ler93,

Did you upload the image file to cognition.run?

**Reply by ler93** [reply]
Ohhh, I forgot to do that! Now I uploaded the image and it worked perfectly. 

Thank you very much to both! Your help is priceless.
